Tasks/Common/VstsAzureHelpers_/InitializeFunctions.ps1

```diff
@@ -83,6 +83,7 @@ function Initialize-AzureSubscription {
             } catch {
                 # Provide an additional, custom, credentials-related error message.
                 Write-VstsTaskError -Message $_.Exception.Message
+                Assert-TlsError -exception $_.Exception
```

Comment:
This will print error message twice. You can use Assert-TlsError to return a wrapped error message and then print here

Reply:
It won't print twice. We don't print original exception in Assert-TlsError.

Reply:
But the original error message will get printed twice.